test/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import trustme
+
+from dummyserver.server import (
+    DEFAULT_CA,
+    DEFAULT_CA_KEY,
+    CLIENT_INTERMEDIATE_PEM,
+    CLIENT_NO_INTERMEDIATE_PEM,
+    CLIENT_INTERMEDIATE_KEY,
+)
+
+
+@pytest.fixture(scope="session")
+def generated_certs(tmp_path_factory):
```

Comment:
Hate to be nit-picky about names but can we call this `certs_dir` or something shorter? :)

Reply:
Fixed, thanks.